docs/Installation-Guide.rst

```diff
@@ -87,7 +87,25 @@ Also you may want to read `gcc Tips <./gcc-Tips.rst>`__.
 macOS
 ~~~~~
 
-LightGBM depends on **OpenMP** for compiling, which isn't supported by Apple Clang.
+LightGBM can use Apple Clang, or GCC/G++ to build in macOS.
+
+Apple Clang
+***********
+
+1. Install `CMake`_ (3.12 or higher).
```

Comment:
It will be nice if the certain version of AppleClang been specified.

Reply:
I am not very certain about the minimal version. I know 
macOS >= el_capitan is supported by https://github.com/Homebrew/homebrew-core/blob/master/Formula/libomp.rb

Reply:
A reasonable guess would be 9.0 (since 10.12).